must-gather/collection-scripts/gather_clusterscoped_resources

```diff
@@ -64,3 +64,20 @@ for command_desc in "${commands_desc[@]}"; do
      COMMAND_OUTPUT_FILE=${BASE_COLLECTION_PATH}/cluster-scoped-resources/oc_output/desc_${command_desc// /_}
     { oc describe "${command_desc}"; } >> "${COMMAND_OUTPUT_FILE}"
 done
+
+kenel_collection() {
+        oc debug nodes/"${node}" --to-namespace="${ns}" -- bash -c "sleep 5m" &
+        while [ "$(oc get pods -n openshift-storage | grep "${node//./}-debug" | awk '{print $2}')" != "1/1" ] ; do
+            sleep 3
+            echo "waiting for the debug pod to be in ready state"
+        done
+        oc rsync -n "${ns}" "$(oc get pods -n "${ns}"| grep "${node//./}-debug"| awk '{print $1}')":/var/log/ "${KERNEL_OUTPUT_DIR}"
+    }
```

Comment:
rajat the logic is fine. I will suggest you to using the cat cmd and work with that because using this approach will cause failures in MG.

Reply:
@crombus Can you provide an example of what you mean?

Reply:
@jarrpa he means to implement it this way
https://github.com/openshift/ocs-operator/blob/master/must-gather/collection-scripts/gather_ceph_resources#L199
Since it's faster and using the `oc rsync` command, we are running into an issue that we discussed offline/below.

The script is not able to differentiate which `debug` pod to collect logs from. And hence, line 72 `" waiting for the debug pod to be in ready state"` keeps on looping infinitely.

We can avoid this issue by using the cat command I think. I have still not implemented it but I think that way, its better based on our disscussion.